moj_auth/api_client.py

```diff
@@ -130,6 +130,28 @@ def token_saver(token, request, user):
     return _get_slumber_connection(session)
 
 
+def get_authenticated_connection(username, password):
+    """
+    Returns:
+        an authenticated slumber connection
+    """
+    session = OAuth2Session(
```

Comment:
Not really in this case - the difference is that `MoJOAuth2Session` raises `Unauthorized` on 401, which is what is caught by the middleware to redirect the user to the login page. The use case of this method is for non-end-user-authenticated requests, and thus we would expect to be checking the response status, handling a 401 as an internal error.